presto-parser/src/main/java/io/prestosql/sql/tree/RefreshMV.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class RefreshMV
+        extends Statement
+{
+    private final QualifiedName target;
+    private final Query query;
```

Comment:
The node should be a reflection of the query syntax. As such, the `Query` has no place here, since it's not something that would be typed as part of the `REFRESH MATERIALIZED VIEW xxx` command. I understand why you added it, but see my comment in RefreshMaterializedViewRewrite for how to approach it.

Reply:
Got it.